v2/objectstore/object_store.go

```diff
@@ -288,23 +288,23 @@ func downloadFile(ctx context.Context, bucket *blob.Bucket, blobFilePath, localF
 
 // The endpoint uses Kubernetes service DNS name with namespace:
 // https://kubernetes.io/docs/concepts/services-networking/service/#dns
-const defaultMinioEndpointInMultiUserMode = "minio-service.kubeflow:9000"
-const minioArtifactSecretName = "mlpipeline-minio-artifact"
+const defaultObjectStoreEndpointInMultiUserMode = "minio-service.kubeflow:9000"
+const objectStoreArtifactSecretName = "mlpipeline-minio-artifact"
 
-func MinioDefaultEndpoint() string {
+func ObjectStoreDefaultEndpoint() string {
 	// Discover minio-service in the same namespace by env var.
 	// https://kubernetes.io/docs/concepts/services-networking/service/#environment-variables
-	minioHost := os.Getenv("MINIO_SERVICE_SERVICE_HOST")
-	minioPort := os.Getenv("MINIO_SERVICE_SERVICE_PORT")
+	minioHost := os.Getenv("OBJECT_STORE_SERVICE_HOST")
```

Comment:
Just wondering, are we still want to change the env variable to `OBJECT_STORE_SERVICE_HOST`? If so we need to set the `OBJECT_STORE_SERVICE_HOST` as part of our manifest. The default object store integration for open source has the `minio-service` name.

Reply:
@rimolive just want to confirm, you want to change the name to `OBJECT_STORE_SERVICE_HOST` for the v2 (next gen) object store only?